automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Clickable.scala

```diff
@@ -20,20 +20,22 @@ trait Clickable extends LazyLogging { this: Component =>
     } catch {
       case e: TimeoutException =>
         // show me query string on failed WebElement
-        new WebDriverWait(webDriver, 5).until(ExpectedConditions.elementToBeClickable(query.element.underlying))
+        new WebDriverWait(webDriver, 1).until(ExpectedConditions.elementToBeClickable(query.element.underlying))
```

Comment:
why reducing the wait time?

Reply:
I didn't see need for longer wait to terminate test. It has no useful purpose. thank you for review.